en/docs/portal/transports/add-transport.md

```diff
@@ -8,27 +8,26 @@ To add a new transport, click the **Add New Transport** button at the top right
-|Transfer Strategy     |<li><strong> Default </strong>: Low cost and high efficiency. </li> <li><strong> Interactive </strong>: For applications requiring better response times, HDT internal routing is enabled in this mode. </li> <li><strong> High Concurrency </strong>：Suitable for small data transfers and high concurrency. Normally, concurrency is considered to be high when its value exceeds 1,000 connections and low when its value is below 100 connections. </li> <li><strong> High Real-time </strong>: Suitable for applications delivering small data with low concurrency (fewer than 100 connections) and requiring optimized response times. </li> |
+| Carry Client IP      | <strong> NO </strong>: Disable the function. By default, HDT will try to add the X-Forwarded-For header into the request to carry the client’s real IP address for HTTP and HTTPS applications. (For HTTPS applications, the client certificate must be deployed on the HDT servers. Please contact the HDT support team if you need to deploy a client certificate.) <br> <strong> TCP Option 0x4e </strong>: HDT establishes a TCP connection with the origin server and sends the client's real IP address to the origin server through the tcp-option 0x4e in the TCP message; the origin server can obtain the client's real IP address through the F5 device or the Linux kernel module provided by HDT. <br> <strong>proxy protocol v1</strong>: By using proxy protocol v1, HDT establishes a TCP connection with the origin server and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v1 message. <br> <strong>proxy protocol v2 </strong>: By using proxy protocol v2, HDT establishes a TCP connection with the origin server and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v2 message.|
```

Comment:
Not sure what "to carry" means. Do you mean "to use"?

Reply:
"to carry" here is to carry the end-user IP info in HTTP extension header so that those info will not lost in the proxying process.

You can find more descriptions in NGINX doc.
https://www.nginx.com/resources/wiki/start/topics/examples/forwarded/

Reply:
The latest version:
| Preserve Client IP Addresses | **NO**: Disable the function.<br>**TCP Option 0x4e**: By using TCP Option 0x4e, HDT sends the client's real IP address to the origin server.<br>**Proxy Protocol v1**: By using Proxy Protocol v1, HDT sends the client's real IP address to the origin server in the first packet.<br>**proxy protocol v2**: By using Proxy Protocol v2, HDT sends the client's real IP address to the origin server in the first packet.|